resources/profiles/AnkerMake.idx

```diff
@@ -0,0 +1,2 @@
+min_slic3r_version = 2.5.0-alpha3
```

Comment:
@ankermake What version of Prusa Slier did you test on? I would use that as the min slicer version